comments.js

```diff
@@ -0,0 +1,66 @@
+import { ADD_COMMENT, EDIT_COMMENT, REMOVE_COMMENT, THUMB_UP_COMMENT, THUMB_DOWN_COMMENT } from './actions';
+
+const initialState = {
+    comments: [],
+    users: []
+};
+
+export const comments = (state = initialState, action) => {
+
+    switch (action.type) {
+
+        case ADD_COMMENT:
+            return [{
+                id: action.id,
+                text: action.text,
+                votesUp: 0,
+                votesDown: 0
+            }, ...state.comments];
+            break;
+
+        case EDIT_COMMENT:
+            const check = state.comments.find(comment => (comment.id === action.id));
+            if (check) {
+                return check.text = action.text;
```

Comment:
dobrze by było zrobić tutaj to samo co w THUMB_UP_COMMENT i THUMB_DOWN_COMMENT czyli zwrócić:
```
{
  ...check
  text: action.text
}
```